pkg/kubelet/qos/policy.go

```diff
@@ -72,8 +72,8 @@ func GetContainerOOMScoreAdjust(pod *v1.Pod, container *v1.Container, memoryCapa
 		return (1000 + guaranteedOOMScoreAdj)
 	}
 	// Give burstable pods a higher chance of survival over besteffort pods.
-	if int(oomScoreAdjust) == besteffortOOMScoreAdj {
-		return int(oomScoreAdjust - 1)
+	if int(oomScoreAdjust) >= besteffortOOMScoreAdj {
+		return int(besteffortOOMScoreAdj - 1)
```

Comment:
Just a small improve. I think `int(oomScoreAdjust) >= besteffortOOMScoreAdj` is more understandable than `int(oomScoreAdjust) == besteffortOOMScoreAdj`. If `besteffortOOMScoreAdj` is changed to another value(e.g. 996, any value less than 1000), `int(oomScoreAdjust) >= besteffortOOMScoreAdj` will still be OK to handle the case.